08-projects.Rmd

```diff
@@ -10133,14 +10123,14 @@ ggmap(map)
 - A description of what your script does.
 ```
 
-##### 7. `fread` is a "Fast and Friendly File Finagler". It is part of the very popular `data.table` package in R. We will learn more about this package next semester. For now, read the documentation [here](https://www.rdocumentation.org/packages/data.table/versions/1.12.8/topics/fread) and use the `cmd` argument in conjunction with your script from (4) to preprocess data prior to reading it into a `data.table` in your R environment. 
+##### 7. You have used `fread` in (2). Now use the `cmd` argument in conjunction with your script from (6) to read the script output into a `data.table` in your R environment. 
```

Comment:
The students have used `fread` in Question 2, and thus I simplified the question description.

Also, according to the solution, I change "(4)" to "(6)".

Reply:
Good catch!